util/LOGGING.md

```diff
@@ -0,0 +1,75 @@
+# LOGGING
+
+
+## Defaults for logging
+
+Use of the standard Go `log` package is regulated through `logging.go` in the `util` package.
+
+By default, **logging is disabled**. Any `log.Printf` statement will simply be discarded.
+
+To enable logging, you should use 
+
+```go
+util.EnableLogging = true
+util.SetLog()
+```
+
+When enabled, the default output for logging is a file named `govcd-API-YYYY-MM-DD.HH.log`.
```

Comment:
I was thinking about the default frequency of the logs generated. We want the defaults to be easy to use and not overload the user. It's now generating a log file each hour, but even if we reduce it to getting a new log each day it's still too noisy for a default.

Please change the default log filename to `go-vcloud-director.log` without date/time reference, but add a write-up that the YYYY-MM-DD, etc. are supported, so the advanced clients know that's available.

BTW, this will make it consistent with the Python vcd CLI default logging.